plugins/woocommerce-admin/client/homescreen/activity-panel/stock/test/index.js

```diff
@@ -62,4 +74,34 @@ describe( 'StockPanel', () => {
 			expect( invalidateResolution ).toHaveBeenCalled();
 		} );
 	} );
+	it( 'should fire tracks event activity_panel_stock_update_stock when Update stock is clicked', async () => {
+		const createNotice = jest.fn();
+		const invalidateResolution = jest.fn();
+		const updateProductStock = jest.fn().mockResolvedValue( true );
+
+		const { getByRole } = render(
+			<StockPanel
+				lowStockProductsCount={ 1 }
+				isError={ false }
+				isRequesting={ false }
+				products={ [
+					{
+						id: 1,
+						name: 'Test Product',
+						low_stock_amount: 5,
+						stock_quantity: 1,
+						type: 'simple',
+					},
+				] }
+				invalidateResolution={ invalidateResolution }
+				updateProductStock={ updateProductStock }
+				createNotice={ createNotice }
+			/>
+		);
+		userEvent.click( getByRole( 'button', { name: 'Update stock' } ) );
+		expect( recordEvent ).toHaveBeenLastCalledWith(
```

Comment:
Should we use `toHaveBeenCalledWith` instead of `toHaveBeenLastCalledWith`? We don't need this Tracks event to be the last one logged, right?